tests/unit/client/data-access/UlsLanguageRepository.spec.ts

```diff
@@ -0,0 +1,40 @@
+import UlsLanguageRepository from '@/client/data-access/UlsLanguageRepository';
+import { StringTMap } from '@/datamodel/LanguageTranslations';
+
+describe( 'UlsLanguageRepository', () => {
+
+	it( 'gets the language name from the given function', () => {
```

Comment:
confusing test description. should mention Wikibase and ULS maybe, not "given function"